tools/IceRpc.Slice.Tools/IceRpc.Slice.Tools.csproj

```diff
@@ -77,7 +78,10 @@
          <Pack>true</Pack>
          <PackagePath>\</PackagePath>
       </None>
-      <None Include="$(IntermediateOutputPath)/$(AssemblyName).dll" Pack="true" PackagePath="tasks/" Visible="false" />
+      <None Include="$(OutputPath)/$(AssemblyName).dll" Pack="true" PackagePath="tasks/" Visible="false" />
+      <None Include="$(OutputPath)/Microsoft.Bcl.AsyncInterfaces.dll" Pack="true" PackagePath="tasks/" Visible="false" />
```

Comment:
I don't understand why we bundle this DLL. It's in case we execute this tool with a very old .NET framework dotnet?

Reply:
Because System.Text.Json depends on it, and if we don't package it Visual Studio 2022 will fail to load our task